tests/qchem/test_basis_set.py

```diff
@@ -397,6 +399,33 @@ def test_atom_basis_data(self, basis_name, atom_name, params_ref):
         assert alpha == alpha_ref
         assert coeff == coeff_ref
 
+    @pytest.mark.parametrize(
```

Comment:
Should we add a couple of simple tests for the molecule and Hamiltonian as well?

Reply:
Included `load_data` flag in a test.